src/decoder/mod.rs

```diff
@@ -1141,9 +1141,10 @@ impl<R: Read + Seek> Decoder<R> {
             CompressionMethod::LZW => {
                 Box::new(LZWReader::new(reader, usize::try_from(compressed_length)?))
             }
-            CompressionMethod::PackBits => {
-                Box::new(PackBitsReader::new(reader, usize::try_from(compressed_length)?)?.1)
-            }
+            CompressionMethod::PackBits => Box::new(PackBitsReader::new(
+                reader,
+                usize::try_from(compressed_length)?,
```

Comment:
This `usize::try_from` can be removed if `new` method is changed to accept `u64` to begin with.